package.json

```diff
@@ -119,6 +120,8 @@
     "@typescript-eslint/parser": "^8.3.0",
     "@vitest/coverage-v8": "^2.0.5",
     "@vitest/ui": "^2.0.5",
+    "benchmark": "^2.1.4",
+    "dom^10.0.0": "link:@testing-library/dom^10.0.0",
```

Comment:
This is for running the benchmark tests. We don't need to keep it in.

Reply:
I would like to consider keeping benchmark files if possible. What will happen if we remove this line?

Reply:
Do you mean removing the dependencies? We could create our own small benchmarking solution. It might be fine for our use cases. Not really sure off the top of my head what it would take.

Reply:
Sorry for the confusion. I'm just curious `^10.0.0` in the name part and the `link:` protocol. Why are they needed?